src/app/controllers/PostsCtrl.ts

```diff
@@ -0,0 +1,145 @@
+import tippy from "tippy.js";
+import 'tippy.js/dist/tippy.css';
+import swal from "sweetalert";
+
+import PostService from "../../core/services/PostService";
```

Comment:
Not sorted alphabetically. Imports need to be sorted alphabetically. After merge of master into your branch you should get the lintner configuration with error highlighting in vs.